src/services/mentees.js

```diff
@@ -1107,9 +1108,6 @@ module.exports = class MenteesHelper {
 				})
 				.filter((value) => value !== null)
 
-			// update count after filters
-			userDetails.data.result.count = userDetails.data.result.count
```

Comment:
@adithyadinesh0412 why this line removed

Reply:
userDetails already have count in it. And the count is not modified. We are using the count from userDetails.
It will be returned in the response.